src/components/RichTextEditor/plugins/checkboxListPlugin/createCheckboxListPlugin.ts

```diff
@@ -1,12 +1,12 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 import { createPluginFactory } from '@udecode/plate';
-import { CheckboxItemElement } from '../../components';
+import { CheckboxListElementTag } from '../../components';
 
 export const ELEMENT_CHECK_ITEM = 'checkbox_item';
```

Comment:
there is snake-case and camel-case in the id's (i.e. AlignCenterPlugin), I'd stick to snake-case

Reply:
This is snake case.

Reply:
Got idea, i need to distinguish between the names and I will do that with the postfix NODE => CheckboxListElementNode

Reply:
Are your suggestion was to do? `'checkbox-item'`

Reply:
ah sorry, yes kebab-case i meant 😉